tools/qontract_cli.py

```diff
@@ -305,7 +305,7 @@ def soaking_str(soaking, upgrade_policy, upgradeable_version):
         mutexes = c.get("conditions", {}).get("mutexes") or []
         item = {
             "cluster": cluster_name,
-            "version": parse_semver(version),
+            "version": version,
```

Comment:
is there a reason you don't parse anymore?

Reply:
there was no need.. and we were converting back to text for print out.